src/components/__tests__/ListItem.spec.tsx

```diff
@@ -14,19 +14,20 @@ describe('ListItem', () => {
   const icon = 'https://example.com'
   const title = 'Discipline Item title'
   const badge = '12'
+  // const arrow = false
 
-  const renderDisciplineItem = (): RenderAPI =>
-    render(<ListItem onPress={onPress} description={description} icon={icon} title={title} badgeLabel={badge} />)
+  const renderDisciplineItem = (arrow:boolean): RenderAPI =>
```

Comment:
please rename to arrowDisabled, also you can define it to be false if no parameter is passed by adding defining it like this `const renderDisciplineItem = (arrowDisabled = false): RenderAPI =>` and then where it is false you can just call it without a parameter.